### PR TITLE
Update GitHub Actions; Fix Root Password Shortness in Tests; Fixture Update Script

### PR DESCRIPTION
**Pull request by zliang-akamai**
1. Fix the tests by increasing root password to satisfy the latest requirements.
2. Implement a script to automatically generate fixtures contains a specific string.
3. Update fixtures with the script.
4. Switch acctest command to GUI trigger.
5. Update legacy GitHub Actions.

### CODE REVIEW

File: ad_hoc/update_fixtures_contains_specific_string.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import sys
```
Comment (lgarber-akamai) [inline ad_hoc/update_fixtures_contains_specific_string.py:1]: We might be able to put this file in the `scripts` directory. Thoughts?

Comment (zliang-akamai) [reply ad_hoc/update_fixtures_contains_specific_string.py]: Moved, thank you for the suggestion!

File: scripts/update_fixtures_contains_specific_string.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import sys
+import subprocess
+from pathlib import Path
+
+repo_dir = Path(__file__).parent.parent
+directory = repo_dir / Path("test/integration/fixtures")
+
+# search_string = "\"root_pass\":\"R34lBAdP455\""
```
Comment (yec-akamai) [inline scripts/update_fixtures_contains_specific_string.py:8]: nit: need to clean it up lol

**Review by yec-akamai — APPROVED** [review]
Great work!

File: scripts/update_fixtures_contains_specific_string.py (hunk 2 above)
Comment (zliang-akamai) [reply scripts/update_fixtures_contains_specific_string.py]: Fixed, thanks!

**Pull request merged**